it/v1/endpoints/AmendPensionsReliefsControllerISpec.scala

```diff
@@ -53,27 +65,27 @@ class AmendPensionsReliefsControllerISpec extends IntegrationBaseSpec {
          |{
          |  "links": [
          |    {
-         |      "href": "/individuals/reliefs/pensions/$nino/$taxYear",
+         |      "href": "/individuals/reliefs/pensions/$nino/$mtdTaxYear",
          |      "method": "GET",
          |      "rel": "self"
          |    },
          |    {
-         |      "href": "/individuals/reliefs/pensions/$nino/$taxYear",
+         |      "href": "/individuals/reliefs/pensions/$nino/$mtdTaxYear",
          |      "method": "PUT",
          |      "rel": "create-and-amend-reliefs-pensions"
          |    },
          |    {
-         |      "href": "/individuals/reliefs/pensions/$nino/$taxYear",
+         |      "href": "/individuals/reliefs/pensions/$nino/$mtdTaxYear",
          |      "method": "DELETE",
          |      "rel": "delete-reliefs-pensions"
          |    }
          |  ]
          |}
          |""".stripMargin)
 
-    def uri: String = s"/pensions/$nino/$taxYear"
+    def uri: String = s"/pensions/$nino/$mtdTaxYear"
 
-    def desUri: String = s"/income-tax/reliefs/pensions/$nino/$taxYear"
+    def downstreamUri: String
```

Comment:
Could maybe move this (def downstreamUri) above to line 47 just to group the abstract methods together?